assignments/arrays.js

```diff
@@ -54,41 +54,159 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
+***********************************************************************************/
+// The car lot manager needs to find out how many cars are older than the 
+// year 2000. Using the carYears array you just created, find out how many 
+// cars were made before the year 2000 by populating the array oldCars and 
+// logging it's length.
 
-// ==== Challenge 5 ====
-// The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars = [];
-console.log(); 
 
-// ==== Challenge 6 ====
-// A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
+for(let i = 0; i < inventory.length; i++) {
+    if(inventory[i].car_year < 2000)
+        oldCars.push(inventory[i].car_year);
+}
+
+logSolution(4, `There are ${oldCars.length} cars older than 2000`);
+
+        /*
+            // alternative solution:
+
+            let numOldCars = carYears.filter(carYear => carYear < 2000).length;
+
+            logSolution(5, `There are ${numOldCars} cars older than 2000`);
+        */
+
+
+/**********************************************************************************
+*                                  CHALLENGE 6                                    *
+***********************************************************************************/
+// A buyer is interested in seeing only BMW and Audi cars within the inventory.  
+// Return an array that only contains BMW and Audi cars.  Once you have populated 
+// the BMWAndAudi array, use JSON.stringify() to show the results of the array in 
+// the console.
+
+
 let BMWAndAudi = [];
-console.log();
+
+for(let i = 0; i < inventory.length; i++) {
+    if(inventory[i].car_make === 'BMW' || inventory[i].car_make === 'Audi')
+        BMWAndAudi.push(inventory[i]);
+}
+
+logSolution(6, JSON.stringify(BMWAndAudi), 'Stringified Results:');
+
+    /*  
+        // alternative solution
+
+        let BMWAndAudi = JSON.stringify(inventory.filter(car => car.car_make === 'BMW' || car.car_make === 'Audi'));
+
+        logSolution(6, JSON.stringify(BMWAndAudi), 'Stringified Results');
+
+    */
 
 
+/**********************************************************************************
+*                                Print Solution  (I got bored)                    *
+***********************************************************************************/
 
+function logSolution(num, solution, message){
+    console.log(`/**********************************************************\n*                         CHALLENGE ${num}                     *\n***********************************************************/`);
+    
+    if(message) 
+        console.log(message + '\n');
```

Comment:
I suggest you use brackets for your if statements. While the syntax you wrote is correct, it's harder to modify, since you'd have to add back the brackets anyways if there's more than one line to deal with. It's also good to clearly define the block scope of the if statement.